MobileBuy/sample/src/main/java/com/shopify/sample/util/Util.java

```diff
@@ -53,7 +59,8 @@ public static String checkNotBlank(String reference, @Nullable Object errorMessa
     return reference;
   }
 
-  @NonNull public static <T, R> List<R> mapItems(@NonNull final Collection<T> source, @NonNull final Function<T, R> transformer) {
+  @NonNull
```

Comment:
strange it supposed to be on the same line with method in code style

Reply:
That is what I notice too. I use the same style in the repo. Hmmmmmmmmm.